src/twisted/names/client.py

```diff
@@ -211,15 +212,19 @@ def pickServer(self):
             return self.dynServers[self.index - serverL]
 
 
-    def _connectedProtocol(self):
+    def _connectedProtocol(self, ipv6=False):
```

Comment:
Optional: make it `interface` and then just pass it through to `listenUDP` as the interface argument - no conditional required here.